src/components/Auth/Buttons/LogoutButton.js

```diff
@@ -0,0 +1,26 @@
+import { useAuth0 } from '@auth0/auth0-react';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { userLogout } from '../../../reduxStore/userSlice';
+
+const LogoutButton = () => {
+  const { logout } = useAuth0();
+  const { dispatch } = useDispatch();
+
+  const handleLogout = () => {
+    // dispatch(userLogout());
+    logout({
+      logoutParams: {
+        returnTo: window.location.origin,
+      },
+    }).then(dispatch(userLogout()));
+  };
+
+  return (
+    <a href="/" onClick={handleLogout}>
```

Comment:
Should use a <Button> tag unless there is a specific reason its not.  see MyCoverCropReset.js